deployment/terraform/sqs.tf

```diff
@@ -0,0 +1,51 @@
+# sqs.tf
+
+resource "aws_sqs_queue" "kf_lambda_sqs" {
+  #checkov:skip=CKV_AWS_27:TODO-Will need a seperate story for adding encryption to SQS
+  #ts:skip=AWS.SQS.NetworkSecurity.High.0570 TODO-Will need a seperate story for adding encryption to SQS
```

Comment:
In it's current state, encrypting SQS breaks the lambda. Adding these skips until we can address that in a separate story.